### PR TITLE
attempt to draw the reader to the section that helps them with their intent

### PR DESCRIPTION
**Pull request by sethladd**
cc @hixie

Reasoning: new users frame their intent as "how do I handle an input/tap?". "Managing state" is a solution to the intent/problem.

### CODE REVIEW

**Comment by Hixie** [comment]
LGTM.

We could go further. P4 was looking for "how to change my widget", so maybe "Handling input by changing widgets" or something?

@abarth may have ideas on how else to phrase this.

File: widgets-intro.md [hunk 1]
```diff
@@ -344,8 +344,8 @@ widgets have
 [`onPressed`](https://docs.flutter.io/flutter/material/RaisedButton-class.html#onPressed)
 callbacks that are triggered when the user taps the widget.
 
-Managing state
---------------
+Managing state and handling input
```
Comment (abarth) [inline widgets-intro.md]: Yeah, we should remove "managing state" from the title.  That's the solution.  Instead, we should state the problem:

"Changing widgets in response to input" ?

Comment (sethladd) [inline widgets-intro.md]: can do!

**Comment by sethladd** [comment]
ptal

File: widgets-intro.md [hunk 2]
```diff
@@ -344,8 +344,8 @@ widgets have
 [`onPressed`](https://docs.flutter.io/flutter/material/RaisedButton-class.html#onPressed)
 callbacks that are triggered when the user taps the widget.
 
-Managing state
---------------
+Changing widgets in response to input
+---------------------------------
```
Comment (Hixie) [inline widgets-intro.md]: missing some hyphens :-)

**Comment by Hixie** [comment]
Still LGTM

**Pull request merged**